test/Microsoft.TemplateEngine.Edge.UnitTests/NuGetInstallerTests.cs

```diff
@@ -0,0 +1,487 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Abstractions.GlobalSettings;
+using Microsoft.TemplateEngine.Abstractions.Installer;
+using Microsoft.TemplateEngine.Edge.Installers.NuGet;
+using Microsoft.TemplateEngine.Edge.UnitTests.Mocks;
+using Microsoft.TemplateEngine.Mocks;
+using Microsoft.TemplateEngine.TestHelper;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.Edge.UnitTests
+{
+    public class NuGetInstallerTests : IClassFixture<PackageManager>
+    {
+        private PackageManager _packageManager;
+        public NuGetInstallerTests(PackageManager packageManager)
+        {
+            _packageManager = packageManager;
+        }
+
+        [Fact]
+        public async Task CanInstall_LocalPackage()
+        {
+            MockInstallerFactory factory = new MockInstallerFactory();
+            MockManagedTemplatesPackageProvider provider = new MockManagedTemplatesPackageProvider();
+            string installPath = TestUtils.CreateTemporaryFolder();
+            EnvironmentSettingsHelper helper = new EnvironmentSettingsHelper();
```

Comment:
EnvironmentSettingsHelper is disposable, and use `CreateTemporaryFolder` on helper to ensure deleting of folders...